### PR TITLE
Add an importer script for help articles

### PR DESCRIPTION
**Pull request by lornajane**
* Install pandoc https://pandoc.org/
* Run this script from the command line, with the URL of the page to import

  python import-help-articles.py https://help.aiven.io/en/articles/5074009-forking-an-aiven-service

### CODE REVIEW

File: utils/import-help-article.py [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+import sys
+from urllib.parse import urlparse
+
+import pypandoc
+from bs4 import BeautifulSoup
+
+if len(sys.argv) < 2:
+    sys.exit("Give the URL with the command")
+
+page_url = sys.argv[1]
```
Comment (SmuliS) [inline utils/import-help-article.py:12]: Not requesting any changes here, so this is more of a FYI. I find that using [argparse](https://docs.python.org/3/library/argparse.html) makes building Python CLI tools super easy and more human friendly. Just wanted to share my thought here :)

**Review by diawarr — APPROVED** [review]
Works fine for me :)

One thing to note is that the pandoc conversion adds pipe characters at least to the start of some list items - this doesn't seem to affect the built output, but just something to keep in mind in case it has any impact at some stage ;)

**Comment by lornajane** [comment]
Can you give an example of where you see the additional character, @diawarr ? I guess we will need a few more HTML transformations as we run into things.

**Comment by diawarr** [comment]
I tested with https://help.aiven.io/en/articles/2843308-migrating-elasticsearch-data-to-aiven and https://help.aiven.io/en/articles/5241419-configuring-java-ssl-to-access-kafka - my guess is that it adds the pipe if there are any subelements within the main <li><p class="no-margin> element, even if it's just a link or a <br/> at the end of the content ;)

File: utils/import-help-article.py (hunk 1 above)
Comment (lornajane) [reply utils/import-help-article.py]: :+1: Thanks for the tip! Take a look at the updated version if you have a minute

**Comment by lornajane** [comment]
@diawarr Do you think the additional pipe character is a showstopper? I believe it's actually the very "correct" behaviour when downconverting from a richer format (html) into something more simple (rst) or presenting any multiline bullet points.

**Comment by diawarr** [comment]
@lornajane - no, I don't see it having any impact on the built output, so there's no need to fix it as such, it was more something that would be worth noting for anyone editing the files, as most of the converted files will need at least some manual editing anyway ;)

**Comment by lornajane** [comment]
Totally agree that most of the content needs rewriting anyway so probably not much will survive unchanged when it's fitted into our preferred content format. Glad to hear that you don't think it's a big problem, I've pushed the docs now so I'll convert to a proper pull request and request your review now all the pieces are there.

**Review by diawarr — APPROVED** [review]

**Pull request merged**